k8s/base/kustomization.yaml

```diff
@@ -0,0 +1,26 @@
+---
+namespace: ceramic-hds
+
+resources:
+  - ipfs.yaml
+  - composedb.yaml
+  - postgres.yaml
+
+configMapGenerator:
+  - name: composedb-template
+    files:
+      - daemon-config.json
+  - name: composedb-env
+    envs:
+      - ./composedb.env
+
+images:
+  - name: 3boxben/composedb
```

Comment:
to check in should we make this in a standard location?

Reply:
The name is a placeholder, but yeah, it should be updated once we publish a docker image with the composedb cli.